app-config.yaml

```diff
@@ -30,7 +30,7 @@ backend:
         # host: 127.0.0.1
     csp:
         connect-src: [ "'self'", 'http:', 'https:' ]
-        img-src: [ "'self'", "*.gov.bc.ca", "data:" ]
+        img-src: [ "'self'", "*.gov.bc.ca", "github.com", "bcgov.github.io", "img.shields.io", "data:" ]
```

Comment:
What images are hosted on the added sites that people want to include in DevHub? I understand the reason for the shields one. Are there similar images from github? Like logos that we want to include in our docs? 

Can the images on bcgov.github.io be moved over?

Reply:
The idea with allowing images from GitHub and bcgov.gihub.io would be to make things easier for team that have existing GH Pages or Wikis content to move into DevHub or, simply to use images that live in Github that might not be their own. Certainly, in the former case, we can recommend that images be co-located with doc content, but forcing this might create additional friction.